main.js

```diff
@@ -4,22 +4,34 @@ const active = require('./lib/active');
 const get = require('./lib/get');
 const dynamos = require('./lib/dynamos');
 const health = require('./lib/health');
+const cache = require('./lib/cache');
 
 let metrics;
 
 exports.health = health.check;
 
 exports.get = fromURL => {
+	if(process.env.USE_CACHE){
```

Comment:
could this option be set via the `init()` method instead of an environment variable?
https://github.com/Financial-Times/n-url-management-api-read-client/blob/master/main.js#L22-L25